java/java-ranger-regression/infusion/prop10/Main.java

```diff
@@ -0,0 +1,77 @@
+
+import org.sosy_lab.sv_benchmarks.Verifier;
+
+public class Main {
+
+    public static void main(String[] args) {
+        InfusionMgrFunctional infusionMgr = new InfusionMgrFunctional();
+
+        B localB = new B();
+
+        DW localDW = new DW();
+
+        infusionMgr.init(localB, localDW);
+
+        TopLevelModeOutputs rtuTlmModeIn = new TopLevelModeOutputs();
+
+        OperatorCommands rtuOpCmdIn = new OperatorCommands();
+
+
+        PatientInputs rtuPatientIn = new PatientInputs();
+
+        ConfigOutputs rtuConfigIn = new ConfigOutputs();
+
+        AlarmOutputs rtuAlarmIn = new AlarmOutputs();
+
+        SystemStatusOutputs rtuSysStatIn = new SystemStatusOutputs();
+
+        InfusionManagerOutputs rtyImOut = new InfusionManagerOutputs();
+
+        if (    (0 <= rtuConfigIn.infusionTotalDuration) &&
+                (0 <= rtuConfigIn.vtbiTotal) &&
+                (0 <= rtuConfigIn.flowRateBasal) &&
+                (0 <= rtuConfigIn.flowRateIntermittentBolus) &&
+                (0 <= rtuConfigIn.durationIntermittentBolus) &&
+                (0 <= rtuConfigIn.intervalIntermittentBolus) &&
+                (0 <= rtuConfigIn.flowRatePatientBolus) &&
+                (0 <= rtuConfigIn.durationPatientBolus) &&
+                (0 <= rtuConfigIn.lockoutPeriodPatientBolus) &&
+                (0 <= rtuConfigIn.maxNumberOfPatientBolus) &&
+                (0 <= rtuConfigIn.flowRateKVO) &&
+                (0 <= rtuConfigIn.enteredReservoirVolume) &&
+                (0 <= rtuConfigIn.Configured) &&
+                (0 <= rtuAlarmIn.highestLevelAlarm) &&
+                (0 <= rtuSysStatIn.volumeInfused) &&
+
+                (rtuConfigIn.infusionTotalDuration <=255) &&
+                (rtuConfigIn.vtbiTotal <=255) &&
+                (rtuConfigIn.flowRateBasal <=255) &&
+                (rtuConfigIn.flowRateIntermittentBolus <=255) &&
+                (rtuConfigIn.durationIntermittentBolus <=255) &&
+                (rtuConfigIn.intervalIntermittentBolus <=255) &&
+                (rtuConfigIn.flowRatePatientBolus <=255) &&
+                (rtuConfigIn.durationPatientBolus <=255) &&
+                (rtuConfigIn.lockoutPeriodPatientBolus <=255) &&
+                (rtuConfigIn.maxNumberOfPatientBolus <=255) &&
+                (rtuConfigIn.flowRateKVO <=255) &&
+                (rtuConfigIn.enteredReservoirVolume <=255) &&
+                (rtuConfigIn.Configured <=255) &&
+                (rtuAlarmIn.highestLevelAlarm <=255) &&
+                (rtuSysStatIn.volumeInfused <=255)
+
+                ) {
+            infusionMgr.INFUSION_MGR_Functional(rtuTlmModeIn, rtuOpCmdIn, rtuPatientIn, rtuConfigIn, rtuAlarmIn, rtuSysStatIn, rtyImOut, localB, localDW);
+
+            boolean checkCondition;
+            boolean checkOutput;
+
+
+            //Prop10: mode_intermittent_implies_infusion_rate_intermittent_bolus
+      checkCondition = rtuTlmModeIn.systemOn && (rtyImOut.currentSystemMode == 3);
```

Comment:
The alignment is odd in this place. This holds for the Main.java classes of all properties in the infusion task. Please revise all of them.